checkstyle-tester/projects-to-test-on.properties

```diff
@@ -13,8 +13,8 @@
 #All details at checkstyle/checkstyle#3033: ModalDialogActivationTest till JDK-8166015 ; 'jxc/8073519/**' not compilable by design ; ', jhsdb/**' - checkstyle do not support unicode identifiers
 #openjdk9|hg|http://hg.openjdk.java.net/jdk9/jdk9/jdk/|default|**/test/java/awt/Focus/ModalDialogActivationTest/ModalDialogActivationTest.java,**/test/javax/xml/bind/jxc/8073519/**,**/test/sun/tools/jhsdb/**
 #openjdk10|hg|http://hg.openjdk.java.net/jdk10/jdk10/jdk/|default|**/test/javax/xml/bind/jxc/8073519/**,**/test/sun/tools/jhsdb/**
```

Comment:
Does it make sense to add https://github.com/openjdk/jdk for the latest work?

Reply:
I like this idea. This way we will know about any issues earlier.

Reply:
Do you want it to be as new line 'openjdk-latest' ? Or instead jdk14? 

To catch something on it we should update main repo CI to use it.

Reply:
I am fine with latest and keep each version separate. This way we can do separate regression on each version. Some people don't even run regression on openjdk either because its too big or because of HG.

Reply:
I am vote for the the new line too.

Reply:
we do not use openjdk in CI testing (most likely because it is huge):
```
✔ ~/java/github/romani/checkstyle [master|✔] 
 $ ag --hidden "openjdk14"
.travis.yml
82:    # spotbugs and pmd (openjdk14)
83:    - jdk: openjdk14
263:    # OpenJDK14 build
264:    - jdk: openjdk14
266:        - DESC="build with OpenJDK14"
270:    # OpenJDK14 verify limited
271:    - jdk: openjdk14
273:        - DESC="verify limited with OpenJDK14"
289:    # OpenJDK14 compile input files with jdk14 specific syntax
290:    - jdk: openjdk14
✔ ~/java/github/romani/checkstyle [master|✔] 
06:23 $ ag --hidden "openjdk11"
.travis.yml
75:    # spotbugs and pmd (openjdk11)
76:    - jdk: openjdk11
249:    # OpenJDK11 verify
250:    - jdk: openjdk11
252:        - DESC="verify with OpenJDK11"
256:    # OpenJDK11 assembly/site
257:    - jdk: openjdk11
259:        - DESC="assembly/site with OpenJDK11"
277:    # OpenJDK11 compile input files with jdk9 specific syntax
278:    - jdk: openjdk11
314:    - jdk: openjdk11

src/xdocs/releasenotes.xml
1274:            javadoc build problem on openjdk11.
```

Reply:
fixed.
@pbludov , please be aware that JDK14 already have a lot of new syntax, we should reuse openjdk for source to test our parser update.